dashboard/Dockerfile

```
@@ -1,26 +1,24 @@
-FROM --platform=$BUILDPLATFORM node:25-slim AS base
+FROM --platform=$BUILDPLATFORM node:24-slim AS base
+
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
 
 WORKDIR /work
 
-# use --force to replace existing yarn v1 binaries
-RUN npm install -g corepack@latest --force
+COPY package.json pnpm-lock.yaml .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)